usaspending_api/disaster/v2/views/object_class/spending.py

```diff
@@ -136,13 +137,13 @@ def build_elasticsearch_result(self, info_buckets: List[dict]) -> List[dict]:
             child.pop("parent_data")
             if result["code"] in temp_results.keys():
                 temp_results[result["code"]] = {
-                    "id": int(result["id"]),
+                    "id": result["id"],
                     "code": result["code"],
                     "description": result["description"],
                     "award_count": temp_results[result["code"]]["award_count"] + result["award_count"],
                     # the count of distinct awards contributing to the totals
-                    "obligation": temp_results[result["code"]]["obligation"] + result["obligation"],
-                    "outlay": temp_results[result["code"]]["outlay"] + result["outlay"],
+                    "obligation": round(temp_results[result["code"]]["obligation"] + result["obligation"], 2),
```

Comment:
Might be better to turn the values to `Decimal` so the additions are exact and no rounding is necessary. Would using Decimal mess up the API response?

Reply:
nope, decimal works fine